swift_browser_ui/api.py

```diff
@@ -192,118 +172,106 @@ async def swift_list_shared_objects(
         )
-        serv = tmp_serv.list(container=request.query['container'])
+        serv = tmp_serv.list(container=request.query["container"])
         [_unpack(i, obj, request) for i in serv]
 
         if not obj:
-            request.app['Log'].debug("Empty list in shared container.")
+            request.app["Log"].debug("Empty list in shared container.")
             raise aiohttp.web.HTTPNotFound()
 
         # Some tools leave unicode nulls to e.g. file hashes. These must be
         # replaced as they break the utf-8 text rendering in browsers for some
         # reason.
         for i in obj:
-            i['hash'] = i['hash'].replace('\u0000', '')
-            if 'content_type' not in i.keys():
-                i['content_type'] = 'binary/octet-stream'
+            i["hash"] = i["hash"].replace("\u0000", "")
+            if "content_type" not in i.keys():
+                i["content_type"] = "binary/octet-stream"
             else:
-                i['content_type'] = i['content_type'].replace('\u0000', '')
+                i["content_type"] = i["content_type"].replace("\u0000", "")
 
         return aiohttp.web.json_response(obj)
 
     except SwiftError:
-        request.app['Log'].error("SwiftError occured "
-                                 "return empty container list.")
+        request.app["Log"].error("SwiftError occured " "return empty container list.")
         return aiohttp.web.json_response(obj)
     except ClientException as e:
-        request.app['Log'].error(e.msg)
+        request.app["Log"].error(e.msg)
         return aiohttp.web.json_response(obj)
     except KeyError:
         # listing is missing; possible broken swift auth
-        request.app['Log'].error("listing is missing; "
-                                 "possible broken swift auth.")
+        request.app["Log"].error("listing is missing; " "possible broken swift auth.")
         return aiohttp.web.json_response(obj)
 
 
-async def swift_download_object(
-        request: aiohttp.web.Request
-) -> aiohttp.web.Response:
+async def swift_download_object(request: aiohttp.web.Request) -> aiohttp.web.Response:
     """Point a user to a temporary pre-signed download URL."""
     session = api_check(request)
-    request.app['Log'].info(
-        'API call for download object from {0}, sess: {1} :: {2}'.format(
+    request.app["Log"].info(
+        "API call for download object from {0}, sess: {1} :: {2}".format(
             request.remote,
             session,
             time.ctime(),
         )
     )
 
-    serv = request.app['Sessions'][session]['ST_conn']
-    sess = request.app['Sessions'][session]['OS_sess']
+    serv = request.app["Sessions"][session]["ST_conn"]
+    sess = request.app["Sessions"][session]["OS_sess"]
 
     temp_url_key = await get_tempurl_key(serv)
-    request.app['Log'].debug(
-        "Using %s as temporary URL key", temp_url_key
-    )
+    request.app["Log"].debug("Using %s as temporary URL key", temp_url_key)
```

Comment:
this is neither f string nor format ... need to standardise , check other places as well.